packages/shader-shaderlab/src/shaders/shadingPBR/Refraction.glsl

```diff
@@ -0,0 +1,48 @@
+#ifndef REFRACTION_INCLUDED
+#define REFRACTION_INCLUDED
+
+#ifdef MATERIAL_ENABLE_SS_REFRACTION 
+
+struct RefractionModel{
+    float dist;         // length of the transmission during refraction through the shape
+    vec3 position;      // out ray position
+    vec3 direction;     // out ray direction
```

Comment:
these variables is not semantics clear, you can use such as `transmissionLength`、`positionExit`、`directionExit`